example/deploy-docs-live-code-contracts.ts

```diff
@@ -99,41 +99,41 @@ async function checkBalances(users: string | any[]) {
 }
 
 async function originateTheContracts() {
-  await checkBalances(users);
+  //await checkBalances(users);
```

Comment:
Not sure if it is intended that the file is commented

Reply:
Fixed.